src/molecules/formfields/DateField/index.js

```diff
@@ -32,12 +33,13 @@ function DateField(props) {
     className,
     label,
     input,
+    meta,
   } = props;
 
   const dateParts = dateUtils.dateStrObj(input.value);
 
   return (
-    <div className={classnames(styles['datefield'], className)}>
+    <div className={classnames(styles['datefield'], toggleClass(meta && meta.active, styles['focused']), className)}>
```

Comment:
instead of extracting this into a helper function, you could use `classnames`:
```
classnames(styles['datefield'], {[styles.focused]: meta && meta.active}, className)
```